SU-HSDO/suhumsci
# shs-105: add color variation for mega menu v1

## Summary
This adds a color variation to the original version of the mega menu.

## Steps to Test
1. On the Traditional site, make sure you are using the original megamenu and not the new version. Also make sure that the masthead color is 'default' and not dark (located under theme settings).
2. View the site at mobile size and verify that level two menu items are inheriting the `tertiary` color. On hover, they use `tertiary-darken-20`.
3. On level three, it should **not** use the tertiary color.
4. View at desktop size. The level two drop down items should have the `tertiary` colors.
5. Go to the theme settings and change the masthead color to dark. View the menu at mobile. It should **not** be using any updated tertiary colors (wouldn't pass accessibility). 
6. Under the theme settings, change the color pairing to something else. Verify it's using that color pairings tertiary colors. 
7. Do the same as above but for the colorful site. 

## Notes
As mentioned above, when the masthead is dark the colors don't pass accessibility, so this will only be for the light version.

## Screenshots
![Screen Shot 2022-10-03 at 10 45 46 AM](https://user-images.githubusercontent.com/8405274/193639865-9359930a-9d4a-4487-b44a-77debc841ede.png)

![Screen Shot 2022-10-03 at 10 45 22 AM](https://user-images.githubusercontent.com/8405274/193639878-0150fb78-48b5-43a1-84a2-1b2c41cef736.png)
